XO/XO.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Union, Tuple
+import random
+
+N = 3
+EMPTY = 'E'
+PLAYERS_SIGNS = ['X', 'O']
+
+
+def get_names() -> List[str]:
+    inputs = []
```

Comment:
Hmm now that I think about it, using list-comprehension you can narrow this function to a one-liner.